Deployment/templates/Deploy.yml

```diff
@@ -140,6 +140,7 @@ jobs:
             inputs:
```

Comment:
This comment doesn't belong to this file but can't put it anywhere else. Can you remove the auto-patch-sdk.yml action please? It's not valid now we don't specify SDK version.